package.json

```diff
@@ -20,15 +25,12 @@
     "split"
   ],
   "scripts": {
-    "test": "tape tests/grapheme_splitter_tests.js"
+    "test": "node tests/grapheme_splitter_tests.js"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/orling/grapheme-splitter.git"
   },
   "bugs": "https://github.com/orling/grapheme-splitter/issues",
-  "dependencies": {},
-  "devDependencies": {
-    "tape": "^4.6.3"
-  }
+  "type": "module"
```

Comment:
this (really, the "main" being ESM instead of CJS) means your code simply can't be used by CJS, or older nodes/browsers.

i implore you to reconsider this; either using a build process or authoring in CJS with an ESM wrapper is much more backwards-compatible.

Reply:
I ended writing my own version of this library from scratch anyway. I don't think this PR will ever be merged into grapheme-splitter.